tedana/tests/test_utils.py

```diff
@@ -81,72 +81,77 @@ def test_make_adaptive_mask():
     data = io.load_data(fnames, n_echos=len(tes))[0]
 
     # Just dropout method
-    mask, masksum = utils.make_adaptive_mask(
+    mask, adaptive_mask = utils.make_adaptive_mask(
         data,
         mask=mask_file,
         threshold=1,
         methods=["dropout"],
     )
 
-    assert mask.shape == masksum.shape == (64350,)
-    assert np.allclose(mask, (masksum >= 1).astype(bool))
+    assert mask.shape == adaptive_mask.shape == (64350,)
+    assert np.allclose(mask, (adaptive_mask >= 1).astype(bool))
     assert mask.sum() == 49376
-    vals, counts = np.unique(masksum, return_counts=True)
+    vals, counts = np.unique(adaptive_mask, return_counts=True)
     assert np.allclose(vals, np.array([0, 1, 2, 3]))
-    assert np.allclose(counts, np.array([14974, 3682, 5128, 40566]))
+    assert np.allclose(counts, np.array([14974, 1816, 4428, 43132]))
```

Comment:
I'm going to test this on more datasets tomorrow, but this is not how I'd expect the output values to change for this PR. I'd expect the value for 3 echos to remain constant since, if all three are in the mask, this change would never identify an earlier echo outside the mask. Instead, this looks like there are **more** voxels with 3 good echoes within this PR. This same direction of change happens with all the tests. Am I missing/confused about something?

Reply:
I believe the change is from [bad, bad, good] (old value = 1, new value = 3), [bad, good, good] (old value = 2, new value = 3), and [good, bad, good] (old value = 2, new value = 3).